src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -209,10 +210,22 @@
                             (flatten-row xs path))))))]
     (into {} (flatten-row row [field-name]))))
 
+(defn- type-finder
+  "Mostly just (type member) but with a bit to suss out strings which are ISO8601 and say that they are datetimes"
+  [member]
+  (let [member-type (type member)]
+    (cond
+      (= member-type java.lang.String)
```

Comment:
```suggestion
      (instance? String member-type)
```

- You don't need to do `java.lang.String` since `java.lang` stuff is imported by default
- `instance?` is more idiomatic here. Even tho it's pretty unlikely that `member-type` would be a _subclass_ of `String` since `String` is `final` I wouldn't bet my house on it. Maybe one customer is using a weird JVM that has a secret impl subclass for `String`. I don't know. Better safe than sorry